### PR TITLE
dpkg updates

### PR DESCRIPTION
**Pull request by petere**

### CODE REVIEW

File: Library/Formula/dpkg.rb [hunk 1]
```diff
@@ -23,30 +23,31 @@ class Dpkg < Formula
...
+  def post_install
+    unless File.exist? etc/"dpkg/origins/dummy"
+      (etc/"dpkg/origins/dummy").write <<-EOS
+Vendor: dummy
+EOS
+    end
+    unless File.exist? etc/"dpkg/origins/default"
+      ln_s "dummy", etc/"dpkg/origins/default"
+    end
+  end
```
Comment (tdsmith) [inline Library/Formula/dpkg.rb]: You can just unconditionally write these files in `install` and the bottling code will do the right thing (they won't be overwritten if they already exist on install).

File: Library/Formula/dpkg.rb [hunk 2]
```diff
@@ -23,30 +23,31 @@ class Dpkg < Formula
...
     # use the system "tar", which will fail because it lacks certain switches.
     ENV["TAR"] = Formula["gnu-tar"].opt_bin/"gtar"
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}",
                           "--disable-dselect",
                           "--disable-linker-optimisations",
-                          "--disable-start-stop-daemon",
-                          "--disable-update-alternatives"
```
Comment (tdsmith) [inline Library/Formula/dpkg.rb]: Was this option removed?

Comment (tdsmith) [inline Library/Formula/dpkg.rb]: Ah, just saw your commit message, :+1:

File: Library/Formula/dpkg.rb [hunk 3]
```diff
@@ -23,30 +23,31 @@ class Dpkg < Formula
...
+                          "--disable-start-stop-daemon"
     system "make"
     system "make", "install"
   end
 
+  def post_install
+    unless File.exist? etc/"dpkg/origins/dummy"
+      (etc/"dpkg/origins/dummy").write <<-EOS
```
Comment (mikemcquaid) [inline Library/Formula/dpkg.rb]: Use `EOS.undent` and indent this more nicely, thanks.

File: Library/Formula/dpkg.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/dpkg.rb]: Or you can use `etc.install` which will do the same.

Comment (petere) [inline Library/Formula/dpkg.rb]: If I move these two things into `def install` and remove the `unless` around them, then installation fails with

    Error: Will not overwrite /usr/local/etc/dpkg/origins/dummy

if the file already exists.

Comment (tdsmith) [inline Library/Formula/dpkg.rb]: Sorry, yes, you'll additionally want to use the `etc.install` method.

Comment (petere) [inline Library/Formula/dpkg.rb]: From the existing examples I can only tell that `etc.install` can install an existing file, but not create a new file from a string or create a symlink.

Comment (mikemcquaid) [inline Library/Formula/dpkg.rb]: You can use `(buildpath/"filename").write "contents"; etc.install "filename"`

**Comment by jacknagel** [comment]
Pulled the parts of this that haven't already been applied elsewhere; will need to bump the revision later.

**Pull request closed without merging**